src/hooks/useDydxClient.tsx

```diff
@@ -57,6 +59,11 @@ const useDydxClientContext = () => {
   const [compositeClient, setCompositeClient] = useState<CompositeClient>();
   const [faucetClient, setFaucetClient] = useState<FaucetClient>();
 
+  // assume there's only one option for indexer endpoints
+  const indexerEndpoints = ENVIRONMENT_CONFIG_MAP[selectedNetwork].endpoints.indexers[0];
+  const indexerConfig = new IndexerConfig(indexerEndpoints.api, indexerEndpoints.socket);
+  const indexerClient = new IndexerClient(indexerConfig);
```

Comment:
Are there any advantages if we store the initialized `indexerClient` in `useState` or memo-ize it?